java/java/src/main/java/org/whispersystems/libsignal/util/ByteUtil.java

```diff
@@ -77,24 +79,22 @@ public static int highBitsToInt(byte value) {
 
   public static byte[] longToByteArray(long value) {
     byte[] bytes = new byte[8];
-    bytes[7] = (byte)value;
-    bytes[6] = (byte)(value >> 8);
-    bytes[5] = (byte)(value >> 16);
-    bytes[4] = (byte)(value >> 24);
-    bytes[3] = (byte)(value >> 32);
-    bytes[2] = (byte)(value >> 40);
-    bytes[1] = (byte)(value >> 48);
-    bytes[0] = (byte)(value >> 56);
+    bytes[7] = (byte) value;
+    bytes[6] = (byte) (value >> 8);
+    bytes[5] = (byte) (value >> 16);
+    bytes[4] = (byte) (value >> 24);
+    bytes[3] = (byte) (value >> 32);
+    bytes[2] = (byte) (value >> 40);
+    bytes[1] = (byte) (value >> 48);
+    bytes[0] = (byte) (value >> 56);
     return bytes;
   }
 
   public static long byteArray5ToLong(byte[] bytes, int offset) {
-    return
-        ((bytes[offset]     & 0xffL) << 32) |
-        ((bytes[offset + 1] & 0xffL) << 24) |
-        ((bytes[offset + 2] & 0xffL) << 16) |
-        ((bytes[offset + 3] & 0xffL) << 8) |
-        ((bytes[offset + 4] & 0xffL));
```

Comment:
If I were going to grumble about specific formatting decisions, this one seems pretty unfortunate.